cg/cli/workflow/mip_dna/base.py

```diff
@@ -230,7 +230,9 @@ def start(context: click.Context, dry_run: bool = False):
             LOG.warning("%s: contains non-dna samples - skipping", case_obj.internal_id)
             continue
 
-        if context.obj["tb"].has_analysis_started(case_id=case_obj.internal_id):
+        has_started, status = context.obj["tb"].has_analysis_started(case_id=case_obj.internal_id)
+        if has_started:
+            LOG.warning("%s: analysis is %s - skipping", case_id, status)
```

Comment:
```suggestion
        if context.obj["tb"].has_analysis_started(case_id=case_obj.internal_id):
            LOG.warning("%s: analysis is %s - skipping", case_id, context.obj["tb"].get_analysis_status(case_id=case_obj.internal_id))
```

I think the latest change became too complex, could we add an `get_analysis_status(caseid) ` method instead?

Reply:
Done